<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlteAbfuellung</class>
 <widget class="QWidget" name="AlteAbfuellung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Vorname Nachname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_adresse">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Musterstr. 10, 71540 Murrhardt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_telefon">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0171-12345678 / 07192-123456</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
